tests/integration_tests/cmd/test_schema.py

```diff
@@ -70,16 +71,20 @@ def test_clean_log(self, class_client: IntegrationInstance):
         version_boundary = get_feature_flag_value(
             class_client, "DEPRECATION_INFO_BOUNDARY"
         )
+        boundary_message = "Deprecated cloud-config provided:"
```

Comment:
Agreed. This pattern repeats itself a couple of times in tests and I'd love to make testing this easier.

> That could probably be folded into verify_clean_boot(), but it makes more sense as a separate function to me.

+1 - Trying to pack that functionality into `verify_clean_boot()`'s signature would add more noise than I think is worth. In cases like this it would be cleaner / simpler to `verify_clean_boot(ignore_deprecations=True)` and then use a helper like you suggest.

Reply:
> I don't think it's a blocker for this PR, but I think there's still more work that can be done here.

Agreed, this is is a rough edge that I'd love to make more ergonomic. I might try to tackle that in a follow-up PR sometime soon.